tools/apoc/apoc.xml

```diff
@@ -0,0 +1,265 @@
+<tool id="apoc" name="APoc" version="1.0">
+    <description>Large-scale identification of similar protein pockets</description>
+    <requirements>
+        <requirement type="package" version="1b16">apoc</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command>
+        <![CDATA[ 
+        #if '$templates_source.template_source_select'=="list"#
+            #for $i, $s in enumerate( $templates_source.template )# 
+                echo ${s.input.file_name} >> templates_path ;
+            #end for#
+            paste -d "\t" templates_path '$templates_source.lt' | awk '{print $1"\t"$3}' > templates_list;
+        #end if#
+
+        #if '$query_source.query_source_select'=="list"#
+            #for $i, $s in enumerate( $query_source.query )# 
+                echo ${s.input.file_name} >> queries_path ;
+            #end for#
+            paste -d "\t" queries_path $query_source.lq | awk '{print $1"\t"$3}' > queries_list;
+        #end if#
+
+        $__tool_directory__/tools/apoc/
```

Comment:
I really meant just `apoc`! When Galaxy resolves the requirement installing the BioConda package, `apoc` will be on the `$PATH`.